palantiri/core/datahandler.py

```diff
@@ -44,9 +45,9 @@ def cleanup(self):
         self.insert_evt.set()
 
     def add_page(self, page):
+        # FIXME; Crawler Id hardcoded
         self.to_dump.append(
-            (page.url, page.source, page.access_datetime,
-             page.response_code, json.dumps(dict(page.headers)))
+            (page.url, page.source, page.access_datetime, 2)
```

Comment:
IMO we should at least have the response code here. I'm okay with not adding the headers, but I think we should keep the status code

Reply:
I was just trying to get the code to work with the existing DB structure here.